src/homepageExperience/components/FeedbackBar.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import {ComponentSize, FlexBox, JustifyContent} from '@influxdata/clockface'
+import {ThumbsDown, ThumbsUp} from './HomepageIcons'
+
+type OwnProps = {
+  wizardEventName: string
+}
+export default class FeedbackBar extends React.Component<OwnProps> {
+  private handleThumbsUpClick = () => {
+    // handle event here
+  }
+  private handleThumbsDownClick = () => {
+    // handle event here
+  }
+  render() {
+    const feedbackBarStyle = {
+      background:
+        'linear-gradient(256.11deg, rgba(147, 148, 255, 0.2) 0%, rgba(81, 60, 198, 0.2) 100%)',
+      paddingLeft: '16px',
+      paddingRight: '16px',
+      maxWidth: '493px',
+    }
```

Comment:
let's move this into the homepageExperience css file since it seems pretty static

Reply:
Done!